server/src/main/java/org/elasticsearch/action/admin/indices/shrink/ResizeRequest.java

```diff
@@ -173,4 +176,23 @@ public void setResizeType(ResizeType type) {
     public ResizeType getResizeType() {
         return type;
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
+        builder.startObject(CreateIndexRequest.SETTINGS.getPreferredName());
```

Comment:
Nit: could we add some empty lines or anonymous blocks around start/end object to make the object creation easier to read?